package.json

```diff
@@ -16,8 +16,8 @@
   },
   "devDependencies": {
     "@iconify/svelte": "^3.0.0",
-    "@sveltejs/adapter-static": "next",
-    "@sveltejs/kit": "next",
+    "@sveltejs/adapter-static": "^1.0.0-next.48",
+    "@sveltejs/kit": "^1.0.0-next.539",
```

Comment:
We might want to leave these dependencies as "next".
I think right now we're using unstable versions.